npm/ng-packs/apps/dev-app/src/app/home/home.component.html

```diff
@@ -10,9 +10,23 @@ <h1>{{ '::Welcome' | abpLocalization }}</h1>
 
     <p class="lead px-lg-5 mx-lg-5">{{ '::LongWelcomeMessage' | abpLocalization }}</p>
 
-    <a *ngIf="!hasLoggedIn" (click)="login()" class="px-4 btn btn-primary ml-1" role="button"
-      ><i class="fa fa-sign-in"></i> {{ 'AbpAccount::Login' | abpLocalization }}</a
+    <button
+      *ngIf="!hasLoggedIn"
+      [disabled]="isLog"
+      (click)="login()"
+      class="px-4 btn btn-primary ml-1"
+      role="button"
     >
+      <i class="fa fa-sign-in"></i> {{ 'AbpAccount::Login' | abpLocalization }}
+      <div
```

Comment:
It covered the requirements but we have a directive that name is "AbpLoading".  You should use that directive for loading the spinner.